app/lib/index.js

```diff
@@ -125,7 +125,7 @@ renderer.onSync('app:get-plugins', () => {
 });
 
 renderer.onSync('app:get-flags', () => {
-  return flags;
+  return flags.getAll();
```

Comment:
No test cases? 🤡

Reply:
What I mean by much more time:
1. Setup tests for index.js
2. Or refactor index.js to extract the listeners and test them with flags.